app/Shop/Orders/Repositories/OrderRepository.php

```diff
@@ -207,4 +209,22 @@ public function buildOrderDetails(Collection $items)
             }
         });
     }
+
+    /**
+     * @return Address $address
+     */
+    public function getAddress() : Address
+    {
+        $address = $this->model->first()->address()->first();
+        return $address;
```

Comment:
I think, this should be `->getAddresses()` and returns a collection. Then, if you want to use it in the controller or in a class, you just need to:

```
$orderRepo = new OrderRepository($order);
$order->address = $orderRepo->getAddresses()->first(); // or use ->last()
```